Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -341,7 +360,7 @@ public virtual void OnValidate()
                     Debug.Log("NetworkManager: added default Transport because there was none yet.");
                 }
 #if UNITY_EDITOR
-                UnityEditor.EditorUtility.SetDirty(gameObject);
+                UnityEditor.Undo.RecordObject(this, "Added default Transport");
```

Comment:
```suggestion
                UnityEditor.Undo.RecordObject(this, "Added default NetworkAuthenticator");
```
brain fart

Reply:
@MrGadget1024 this still says Transport

Reply:
nevermind, I just noticed that this is the one for the Transport. we need to fix this in a separate PR.